src/blocks/cpu.rs

```diff
@@ -206,3 +217,15 @@ impl Block for Cpu {
         self.id
     }
 }
+
+/// Read the cpu turbo boost status from kernel sys interface
+/// or intel pstate interface
+fn boost_status() -> Option<bool> {
+    if let Ok(boost) = fs::read_to_string("/sys/devices/system/cpu/cpufreq/boost") {
+        return Some(boost.starts_with("1"));
+    } else if let Ok(no_turbo) = fs::read_to_string("/sys/devices/system/cpu/intel_pstate/no_turbo")
```

Comment:
and also remove the "fs::" here

Reply:
Yeah... If You think so. At beginning I just use read_to_string, but for readability, I thought the fs:: prefix more helpful. You immediately see where this function from and what it do - read string from file system.